src/main/java/org/zaproxy/zest/core/v1/ZestClientElementMouseOver.java

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
@psiinon @thc202 should we be using a different header now?

Reply:
I'm fine either way.

Reply:
I guess thats a separate discussion - I dont mind too much either way either 😉